From 118e814a76032ed5623d9eaa1d5556eb78c0fcab Mon Sep 17 00:00:00 2001
From: Lambert Sartory <lambert.sartory@gmail.com>
Date: Sat, 31 Dec 2022 14:39:12 +0100
Subject: [PATCH] fix RK3588 video output

diff --git a/src/video/rk.c b/src/video/rk.c
index 473512b..989ded6 100644
--- a/src/video/rk.c
+++ b/src/video/rk.c
@@ -346,7 +346,7 @@ int rk_setup(int videoFormat, int width, int height, int redrawRate, void* conte
         if (!prop) {
           continue;
         }
-        if (!strcmp(prop->name, "type") && props->prop_values[j] == DRM_PLANE_TYPE_OVERLAY) {
+        if (!strcmp(prop->name, "type") && props->prop_values[j] == DRM_PLANE_TYPE_PRIMARY) {
           plane_id = ovr->plane_id;
         }
         drmModeFreeProperty(prop);
-- 
2.38.2

